app/src/main/java/woowacourse/shopping/data/di/component/DiSingletonComponent.kt

```diff
@@ -0,0 +1,18 @@
+package woowacourse.shopping.data.di.component
+
+object DiSingletonComponent {
+    private val binds: MutableMap<Class<*>, Any> = mutableMapOf()
+    private const val ERROR_DI_MATCH = "No binding Singleton Component %s"
+
+    fun <T : Any> bind(
+        bindClassType: Class<T>,
+        bindClass: Any,
+    ) {
+        binds[bindClassType] = bindClass
+    }
```

Comment:
bindClass 파라미터가 `bindClassType`객체라는 메세지를 던져 주는 것도 좋을 것 같습니다
```suggestion
    fun <T : Any> bind(
        bindClassType: Class<T>,
        bindClass: T,
    ) {
        binds[bindClassType] = bindClass
    }
```

Reply:
그렇네요.
수정하겠습니다!